### PR TITLE
[BACKLOG-34346] PDI - Not able to connect from a SQL Client tool - Pa…

### PR DESCRIPTION
**Pull request by tkafalas**
…sswordEncoderException

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 53s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/driver
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: assemblies/driver/src/assembly/assembly.xml [hunk 1]
```diff
@@ -30,6 +30,10 @@
         <include>README.txt</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>src/resources</directory>
+      <outputDirectory>.</outputDirectory>
+    </fileSet>
```
Comment (mkambol) [inline assemblies/driver/src/assembly/assembly.xml:36]: Which resources is this bringing in, and what does a user need to do for configuration?

Comment (mkambol) [reply assemblies/driver/src/assembly/assembly.xml:36]: Oh sorry, I see the new resource below.  That will need to be on the classpath, right?  Could/should we place the encoder-plugin.xml in the same directory in the zip as the driver jars?

Comment (tkafalas) [reply assemblies/driver/src/assembly/assembly.xml:36]: I don't see any reason to put it another folder.  It just makes the instructions of how to install more complicated.

**Pull request merged**